src/z3c/breadcrumb/tests.py

```diff
@@ -51,24 +51,26 @@ def doctest_GenericBreadcrumb_interface():
         >>> verifyObject(interfaces.IBreadcrumb, breadcrumb)
         True
 
-    """
+    """  # noqa: F821 undefined name 'rootFolder'
```

Comment:
flake8 allows you to say what extra names get put into doctest globals via the setUp.  You do this by adding

```
[flake8]
builtins = rootFolder
```

This should be doable via [flake8] additional-config in .meta.toml?

Reply:
Fixed in https://github.com/zopefoundation/z3c.breadcrumb/pull/2/commits/6eac84f495373dbf07f84b4ae6af634e9dfbb165.